src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml.cs

```diff
@@ -2,21 +2,48 @@
 // The Microsoft Corporation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.PowerToys.Settings.UI.ViewModels;
+using System;
+using Microsoft.PowerToys.Settings.UI.Lib.ViewModels;
+using Windows.UI.Popups;
+using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 
 namespace Microsoft.PowerToys.Settings.UI.Views
 {
     public sealed partial class ImageResizerPage : Page
     {
-        private ImageResizerViewModel ViewModel { get; set; }
+        public ImageResizerViewModel ViewModel { get; set; }
 
         public ImageResizerPage()
         {
             InitializeComponent();
 
-            ViewModel = new ImageResizerViewModel();
+            ViewModel = new ImageResizerViewModel(ShellPage.SendDefaultIPCMessage);
             DataContext = ViewModel;
         }
+
+        public void DeleteCustomSize(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                Button deleteRowButton = (Button)sender;
+                int rowNum = int.Parse(deleteRowButton.CommandParameter.ToString());
+                ViewModel.DeleteImageSize(rowNum);
+            }
+            catch (Exception exp)
+            {
+            }
+        }
+
+        private void AddSizeButton_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                ViewModel.AddRow();
+            }
+            catch (Exception exp)
+            {
+            }
```

Comment:
Same comment as as above.  We shouldn't fail silently.  What is the exception and what is the best way to handle it?  How does it affect the user, and can they do anything to mitigate it?

Reply:
I have no specific exception in mind, I will add an error pop-up message asking the user to report the failure on Github and log telemetry too.

Reply:
@crutkas. Is this what we want?  I know that we inherited some of this behavior from wox, but despite the fact that it can be quite helpful, I'm personally not a huge fan of the practice of displaying exception messages.    I would rather use telemetry for this.

Reply:
IMO this feels odd since we are catching stuff but not knowing what is failing or why.  Typically try catches would be around known cases for stuff we just can't work around.

Reply:
I removed the exceptions here: https://github.com/microsoft/PowerToys/pull/5765/commits/add35f37069a3ccc808656ae9ab594de09f1f37b